### ISSUE TITLE
about hand size and  data augmentation

### ISSUE THREAD

**Issue by yaoxinreaps** [xinghaochen/Pose-REN#5]
hi, 
thanks for sharing your project, I have two questions here,
1. will the cube size of hand influence the result? for example, change it from 300 to 250, how will the predict result be? have you done some experiments for comparison?
2. which is the better way for data augmentation? just augment for every sample or choose some of them (for example, half probability)?
thank you!

**Comment by xinghaochen** [comment]
Hi, 
1. We didn't conduct extensive experiments for the cube size of the cropped hand. Changing it from 300 to 250 should give similar performance, I guess.
2. We did online data augmentation in training, that is, augmenting each training sample for random transformations before feeding it for training.

**Comment by yaoxinreaps** [comment]
OK, thank you~

**Comment by yaoxinreaps** [comment]
Sorry to bother you again.
Would you please tell me the training parameters on hands17 dataset? More specifically, what is the batch size, initial learning rate, learning policy and stepsize or stepvalue, and max training epoch?
By the way, have you trained the REN model on hands17 dataset for comparison? I guess it can achieve the similar subjective performance because the training data is big enough. What's your opinion?
Thanks again.

**Comment by xinghaochen** [comment]
Hi, here are the training parameters on hands17 dataset:

> base_lr: 0.001
lr_policy: "step"
gamma: 0.1
stepsize: 200000
display: 100
max_iter: 800000
momentum: 0.9
weight_decay: 0.0005

The batch size is 128, the same with all other experiments on our paper.

We did try to triain the REN model on hands17 dataset. Now I can't remember what exactly the performance was. However, Pose-REN did consistently perform better than REN (for several millimeters, maybe) as far as I can remember. 

In case you are now working on the HANDS17 challenge, it should be noted that the pre-trained models on hands17 dataset released in this repo is not exactly the one for our final entry in HANDS17 Challenge. As stated in the challenge document, we used REN as the Init-Net and there were also several improvements/tricks. The pre-trained models on hands17 dataset  in this repo are not intended to reproduce our performance in the challenge, but to provide a more stable and accurate hand pose estimator for various applications.
